### DISCUSSION TITLE
Offline experiment - saving data

### DISCUSSION THREAD

**Discussion by andreaar5** [jspsych/jsPsych#2094] [Q&A]
Hi, 

I am quite new here. 
This year, I run an experiment and I used a free server to save data. 
Now I want to run another experiment offline. 
How can I save data on my laptop? Is it possible? 

Thanks, 
Andrea

**Comment by GEJ1** [comment]
Hi @andreaar5 ,

You can save data on your laptot using 

```javascript 
// first argument is the format, second is the filename.
// the format can be either 'csv' or 'json'.
jsPsych.data.get().localSave('csv','mydata.csv');
```

Check the docs: https://www.jspsych.org/core_library/jspsych-data/#localsave

Best,

Gustavo.

**Comment by jodeleeuw** [comment]
I'm going to shift this to [discussions](https://github.com/jspsych/jsPsych/discussions) since it is a support request.
